recipe/meta.yaml

```
@@ -19,14 +19,22 @@ requirements:
     - pip
     - python >=3.6
   run:
+    - python >=3.6
     - nltk
     - numpy
-    - python >=3.6
     - scikit-learn
     - scipy
-    - pytorch >=1.0.1
     - tqdm
-    - transformers >=3.1
+    - pytorch >=1.6.0
+    - torchvision
+    - sentencepiece
+    - tokenizers >=0.10.3
+    - transformers >=4.6.0,<5.0.0
+  run_constrained:
+    # NOTE: huggingface-hub is not yet available on conda-forge.
+    # Moving huggingface-hub under "run_constrained" as a temporary 
+    # fix, until the package is made available on conda-forge.
+    - huggingface-hub
```

```suggestion
huggingface_hub
```

The package is on conda with `_` instead of `-`.

---

Good catch! Thanks.